packages/lib-classifier/src/plugins/tasks/DrawingTask/components/DrawingTask.js

```diff
@@ -62,16 +49,17 @@ class DrawingTask extends React.Component {
         </StyledText>
 
         {task.tools.map((tool, index) => {
```

Comment:
It's possible that a drawing task can be created, but with no tools defined, so there needs to be an existential check on the line above before we call map.

Reply:
`task.tools` defaults to an empty array.